PennMobile/src/main/java/com/pennapps/labs/pennmobile/LaundryFragment.java

```diff
@@ -39,12 +43,25 @@
     @Bind(R.id.no_results) TextView no_results;
     SwipeRefreshLayout swipeRefreshLayout;
 
+    private SharedPreferences sharedPref;
+
+    @Override
+    public void onStart(){
+        super.onStart();
+    }
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         mLabs = MainActivity.getLabsInstance();
         mActivity = (MainActivity) getActivity();
         mActivity.closeKeyboard();
+
+        sharedPref = getActivity().getPreferences(Context.MODE_PRIVATE);
+    }
+
+    public SharedPreferences getPref(){
+        return sharedPref;
     }
```

Comment:
Why do we have this function?  Line 101 we have a call to shared preference again as well.